pages/runtime/data-api.md

```diff
@@ -0,0 +1,672 @@
+# Data API Service
+
+## Introduction
+Resin's Data API Service constitutes a central node in resin architecture, since it provides interconnection between resin services and communication with the database. At the same time, it exposes an HTTP interface, based on Open Data ([OData](http://www.odata.org)) format, that provides a means to the user to interact with resin resources and fetch or update information regarding their applications, devices and environment variables.
+
+This document aims to describe the endpoints that are provided by the Data API, as well as showing examples of their expected usage.
+
+The requests are expected to follow the OData format, and the responses are provided in JSON format. The authorization token that is required in each request can be found under 'Preferences' in the user's dashboard.
+
+### Base URL
+The base URL for accessing the Data API is:
+
+`https://api.resin.io/v1/`
```

Comment:
Or that, indeed. Then it can be used like {{ `$links.apiBase` }} (or
whatever you call it there).

On Tue, May 24, 2016 at 11:52 AM Shaun Mulligan <notifications@github.com>
wrote:

> In pages/runtime/data-api.md
> <https://github.com/resin-io/docs/pull/240#discussion_r64351609>:
>
> > @@ -0,0 +1,672 @@
> > +# Data API Service
> > +
> > +## Introduction
> > +Resin's Data API Service constitutes a central node in resin architecture, since it provides interconnection between resin services and communication with the database. At the same time, it exposes an HTTP interface, based on Open Data ([OData](http://www.odata.org)) format, that provides a means to the user to interact with resin resources and fetch or update information regarding their applications, devices and environment variables.
> > +
> > +This document aims to describe the endpoints that are provided by the Data API, as well as showing examples of their expected usage.
> > +
> > +The requests are expected to follow the OData format, and the responses are provided in JSON format. The authorization token that is required in each request can be found under 'Preferences' in the user's dashboard.
> > +
> > +### Base URL
> > +The base URL for accessing the Data API is:
> > +
> > +`https://api.resin.io/v1/` <https://api.resin.io/v1/>
>
> may be worth adding it in the links.json?
>
> —
> You are receiving this because you commented.
>
>
> Reply to this email directly or view it on GitHub
> <https://github.com/resin-io/docs/pull/240/files/85d9b4409c26e04d1e868267e1fb47ace85b99b5#r64351609>
>